packages/api-explorer/src/scenes/DiffScene/DocDiff/DocDiff.spec.tsx

```diff
@@ -109,4 +109,61 @@ describe('DocDiff', () => {
       ).toBeInTheDocument()
     })
   })
+
+  it('paginates with correct number of entries per page', async () => {
+    const delta = diffSpecs(leftApi, rightApi, standardDiffToggles)
+    const pageSize = 5
+    renderWithReduxProvider(
+      <DocDiff
+        leftKey={leftKey}
+        leftSpec={leftApi}
+        rightKey={rightKey}
+        rightSpec={rightApi}
+        delta={delta}
+        pageSize={pageSize}
+      />
+    )
+
+    for (let i = 0; i < pageSize; i++) {
```

Comment:
nit: our TS guide suggests using `for..of` or `forEach` for looping over iterables.
```suggestion
    for (row of delta) {
```

Reply:
actually, I see how this is necessary given the parameters.